qutebrowser/misc/sql.py

```diff
@@ -21,12 +21,59 @@
 
 import collections
 
+import attr
 from PyQt5.QtCore import QObject, pyqtSignal
 from PyQt5.QtSql import QSqlDatabase, QSqlQuery, QSqlError
 
 from qutebrowser.utils import log, debug
 
 
+@attr.s
+class UserVersion:
+
+    """The version of data stored in the history database.
+
+    When we originally started using user_version, we only used it to signify that the
+    completion database should be regenerated. However, sometimes there are
+    backwards-incompatible changes.
+
+    Instead, we now (ab)use the fact that the user_version in sqlite is a 32-bit integer
```

Comment:
Do you actually have a major change on the radar, or is this just future-proofing?